node/runtime/pangolin/polkadot-compatible-types.json

```diff
@@ -186,7 +186,7 @@
 			"Erc20": "TokenInfo"
 		}
 	},
-	"BridgeMessageId": "[u8; 16; BridgeMessageId]",
+	"TokenMessageId": "[u8; 16; TokenMessageId]",
```

Comment:
I am curious about this writing style, Can we change it to `"TokenMessageId": "[u8; 16]",`?